### ISSUE TITLE
Training data

### ISSUE THREAD

**Issue by mg515** [isi-vista/structure_via_consensus#5]
Hello!

Thanks for the nice paper and repository! I was wondering, if perhaps you shared your pre-processed training data (occlusion mask, face mask, background mask split), or perhaps the code to preprocess the 3 training datasets mentioned in the paper? Would you be able to do so?

Thanks!

**Comment by iacopomasi** [comment]
Hi @mg515 ,

thank you for the interest in the project. I thought about releasing the process for generating the pseudo labels but it was too complex to bring all together online since depended on diverse modules. I still have in my TODOs to upload the weights for the occlusion detection (see #1 ). Recently, I also moved to another institution so I have less and less time to work on this. I apologize.

Yours,
